install_files/ansible-base/roles/tor-hidden-services/templates/client_auth_app_ssh.j2

```diff
@@ -0,0 +1 @@
+descriptor:x25519:{{ tor_v3_app["app_ssh_key"] }}
```

Comment:
As above, these templates don't need to be unique per-host—see for reference the existing v2 logic:

https://github.com/freedomofpress/securedrop/blob/20cbd52cb40ce1cef09407f81fc338bf6765aaf9/install_files/ansible-base/roles/restrict-direct-access/tasks/fetch_tor_config.yml#L21-L31